apps/roam/src/utils/discourseConfigRef.ts

```diff
@@ -21,6 +21,18 @@ const configTreeRef: {
   nodes: { [uid: string]: { text: string; children: RoamBasicNode[] } };
 } = { tree: [], nodes: {} };
 
+const listeners = new Set<() => void>();
+
+export const subscribe = (listener: () => void) => {
```

Comment:
nice

Reply:
Yeah I spent some time looking through what would be the right solution, how to pass the around and this seemed the right one, I was not sure though but now I am :)